src/crab/ebpf_transformer.cpp

```
@@ -1176,6 +1177,48 @@ static int _movsx_bits(const Bin::Op op) {
+    for (int reg = R0_RETURN_VALUE; reg <= R10_STACK_POINTER; reg++) {
+        const Variable candidate = reg_pack(reg).uvalue;
+        if (candidate == self) {
+            continue;
+        }
+        if (state.values.entail(value <= candidate) && std::ranges::find(variables, candidate) == variables.end()) {
+            variables.push_back(candidate);
+        }
+    }
+}
+
+static std::vector<Variable> live_register_uvalue_upper_bounds_for_unsigned_and_lhs(const TypeToNumDomain& state,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)